openml/flows/flow.py

```diff
@@ -105,7 +377,7 @@ def _ensure_flow_exists(self):
         """
         import sklearn
         flow_version = 'sklearn_' + sklearn.__version__
-        _, _, flow_id = _check_flow_exists(self._get_name(), flow_version)
+        _, _, flow_id = _check_flow_exists(self.name, flow_version)
```

Comment:
This is indeed a bug, but on trying to write a test which covers this function I uncovered a bug on OpenML...